docs/content/en/docs/concepts/architecture/components/scheduler/_index.md

```diff
@@ -14,3 +14,152 @@ The Lifecycle Scheduler can trigger events such as
 deployment, testing, and remediation at specific times or intervals.
 The Keptn Scheduler guarantees that Pods are initiated only after
 the Pre-Deployment checks are completed.
+
+## Architecture of Keptn Scheduler
+
+To integrate the `Keptn Lifecycle Scheduler` into a Kubernetes cluster,
+a Kubernetes manifest is applied with specific **Keptn annotations**.
+These annotations identify the workload and specify the
+pre-deployment and post-deployment tasks and evaluations
+associated with it.
```

Comment:
This is correct more or less, but I would rephrase: if a workload/deployment is annotated with Keptn annotations the webhook will edit it to refer to the Keptn scheduler instead of the default k8s one